assemblies/nexus-base-template/src/main/resources/overlay/etc/ehcache.xml

```diff
@@ -1,32 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ehcache xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://ehcache.org/ehcache.xsd"
-         updateCheck="false"
```

Comment:
For reference, the ehcache3 schema can be found at http://ehcache.github.io/docs/user/3.0/xsds.html